atlas/AtlasMl/atlasml/ml/MLPipelines/PipelineWorkflows.py

```
@@ -0,0 +1,278 @@
+                "cluster_id": str(index)
+                }
+            self.weaviate_client.add_embeddings(CollectionNames.CLUSTERCENTER.value, medoids[index].tolist(), cluster)
+
+        competencies = self.weaviate_client.get_all_embeddings(CollectionNames.COMPETENCY.value)
+        clusters = self.weaviate_client.get_all_embeddings(CollectionNames.CLUSTERCENTER.value)
+
+        for competency in competencies:
+            competency_id, embedding = competency["properties"]["competency_id"], competency["vector"]["default"]
+            similarity_score = np.array([compute_cosine_similarity(embedding, medoid) for medoid in medoids])
+            best_medoid_idx = int(np.argmax(similarity_score))
+            if best_medoid_idx == -1: continue
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)